recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ about:
   home: https://github.com/datamade/probableparsing
   license: MIT
   license_family: MIT
-  license_file: LICENSE
+  license_url: https://raw.githubusercontent.com/datamade/probableparsing/master/LICENSE
```

Comment:
why not just use the github.com URL? you don't need to download the license file, just refer to it

Reply:
unless there's some reason...

Reply:
I had in mind that `conda-build` needed the raw file, not just a link to the webpage

Reply:
fixed